packages/i18nify-js/src/modules/currency/convertToMinorUnit.ts

```diff
@@ -31,7 +31,9 @@ const convertToMinorUnit = (
   const minorUnitMultiplier =
     Math.pow(10, Number(currencyInfo.minor_unit)) || 100;
 
-  const lowerCurrencyValue = amount * minorUnitMultiplier;
+  const lowerCurrencyValue = parseFloat(
+    (amount * minorUnitMultiplier).toFixed(Number(currencyInfo.minor_unit)),
+  );
```

Comment:
since we are converting this to the lowest unit of currency, `toFixed()` can be used which would fallback to `toFixed(0)`. having`.toFixed(Number(currencyInfo.minor_unit))` might not be the way to go